website_multi_theme/models/website_theme.py

```diff
@@ -32,20 +32,34 @@ class WebsiteTheme(models.Model):
              "in websites that enable this theme in multiwebsite mode.",
     )
 
+    dependency_ids = fields.Many2many(
+        'ir.module.module',
+        string="Dependencies",
+        help="Theme-like dependencies. "
+        "Add modules here if you got error \"The style compilation failed\"."
```

Comment:
So, if I get your point, this new field would define other addons which would become part of this "theme", and thus their asset views would become single-website too. Am I right? 🤔 

If you faced this problem on any of the supported themes, then I think you should add this definition into the corresponding `data/themes_*.xml` file.

Also in `demo/themes.xml` would be a way of demoing this new feature too?

Reply:
You understand me correctly.

The problem of this PR is that there is no easy way to check it. All official modules works without those updates. So, seems that the only way to demonstrate it is to create new dummy theme with specific structure